tests/index.test.js

```diff
@@ -46,6 +47,15 @@ test('should grab username and access key from env variable', () => {
         .toContain('key: \'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXfoobar\'')
 })
 
+test('should grab proxy from env variable', () => {
```

Comment:
given this is a simple test, let's also have an https example

Reply:
added https example